hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/jdbc/AllTypesInsertJdbcSqlConnectorTest.java

```diff
@@ -106,7 +108,12 @@ public void insertRowWithAllTypes() throws Exception {
         execute("INSERT INTO " + mappingName + " VALUES(0, " + sqlValue + ")");
         execute("INSERT INTO " + mappingName + " VALUES(1, ?)", javaValue);
 
+        List<Class<?>> columnTypes = new ArrayList<>();
+        columnTypes.add(Integer.class);
+        columnTypes.add(jdbcValue.getClass());
+
         assertJdbcRowsAnyOrder(tableName,
+                columnTypes,
```

Comment:
```suggestion
        assertJdbcRowsAnyOrder(tableName,
                newArrayList(Integer.class, jdbcValue.getClass()),
```

Reply:
Where is newArrayList from? I couldn't resolve what to import.

Reply:
Even better - `List.of` (where list is java.util.List)

Reply:
And end up with compilation failure when you backport :-)